cron/hourly/10-pay-paypal-scheduled-expenses-payouts.js

```diff
@@ -21,7 +21,7 @@ export async function run() {
       { model: models.PayoutMethod, as: 'PayoutMethod', where: { type: PayoutMethodTypes.PAYPAL } },
     ],
   });
-  const batches = values(groupBy(expenses, 'CollectiveId'));
+  const batches = flatten(values(groupBy(values(groupBy(expenses, 'CollectiveId'), 'currency'))));
```

Comment:
This helper's chain is a bit difficult to read (we need a pipe operator in JS!). I'd suggest either splitting into variables, or adding a small comment above like

```suggestion
  // Group expenses by collective and currency
  const batches = flatten(values(groupBy(values(groupBy(expenses, 'CollectiveId'), 'currency'))));
```